src/main/java/com/mparticle/kits/RokoMobiKit.java

```diff
@@ -0,0 +1,260 @@
+package com.mparticle.kits;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+
+import com.mparticle.DeepLinkError;
+import com.mparticle.DeepLinkResult;
+import com.mparticle.MPEvent;
+import com.mparticle.MParticle;
+import com.rokolabs.sdk.RokoMobi;
+import com.rokolabs.sdk.account.RokoAccount;
+import com.rokolabs.sdk.analytics.Event;
+import com.rokolabs.sdk.analytics.RokoLogger;
+import com.rokolabs.sdk.instabot.Instabot;
+import com.rokolabs.sdk.json.Json;
+import com.rokolabs.sdk.links.ResponseVanityLink;
+import com.rokolabs.sdk.links.RokoLinks;
+import com.rokolabs.sdk.links.RokoLinks.CallbackVanityLink;
+import com.rokolabs.sdk.push.PushData;
+import com.rokolabs.sdk.push.PushNotificationHelper;
+import com.rokolabs.sdk.push.RokoPush;
+import com.rokolabs.sdk.push.RokoPushConstants;
+import com.rokolabs.sdk.rokomobi.Settings;
+import com.rokolabs.sdk.tools.ThreadUtils;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.List;
+import java.util.Map;
+
+public class RokoMobiKit extends KitIntegration implements KitIntegration.EventListener, KitIntegration.PushListener, KitIntegration.AttributeListener, KitIntegration.ActivityListener, RokoMobiProvider {
+
+    public static final String INSTABOT_ACTION = "com.rokolabs.mobi.INSTABOT_ACTION";
+    public static final String KIT_NAME = "RokoMobi";
+    public static final String RK_MSG_ID = "rkMsgId";
+    public static final String DEVICE_TOKEN = "deviceToken";
+    public static final String PUSH_ACTION = "com.rokolabs.mobi.PUSH_ACTION";
+
+    @Override
+    protected List<ReportingMessage> onKitCreate(Map<String, String> settings, Context context) {
+        RokoMobi.start(context, new RokoMobi.CallbackStart() {
```

Comment:
you're not using any api key here, as you are on iOS. I think that needs to be corrected.

Reply:
yes, we'll update PR to include a new method to pass in the Instabot API key for SDK initialization from the mParticle wrapper so it can be set server-side.